timber/build.gradle

```diff
@@ -35,8 +35,8 @@ dependencies {
   testCompile deps.festassert
   testCompile deps.junit
   testCompile deps.robolectric
-
   lintChecks project(path: ':timber-lint', configuration: 'lintChecks')
+  compile files('lib/jetbrains-annotations.jar')
```

Comment:
This should also be using the `provided` configuration. And can you put it above all the `testCompile` dependencies with a newline between them?